src/org/usfirst/frc/team2706/robot/subsystems/Camera.java

```diff
@@ -15,178 +14,31 @@
-                            System.out.println("Target found at: " + pr.ctrX + "," + pr.ctrY
-                                            + ", and aspectRatio and boundingArea is: "
-                                            + pr.aspectRatio + "," + pr.boundingArea);
-
-                        prList.add(pr);
-                    } catch (NumberFormatException e) {
-                        continue;
-                    }
-                }
-                // }
-            } catch (java.io.EOFException e) {
-                System.out.println("Camera: Communication Error");
-            }
-
-            sock.close();
-        } catch (UnknownHostException e) {
-            System.out.println("Host unknown: " + RPi_addr);
-            return null;
-        } catch (java.net.ConnectException e) {
-            System.out.println("Camera initialization failed at: " + RPi_addr);
-            return null;
-        } catch (IOException e) {
-            e.printStackTrace();
-            return null;
-        }
-        if (PRINT_STUFF)
-            System.out.println("Network call successful, returning not null data...");
-
-        return prList;
-    }
-
+    
     @Override
     protected void initDefaultCommand() {
-
-    }
-
-    public static final int LEFT_TARGET = -3;
-    public static final int CENTER_TARGET = -1;
-    public static final int RIGHT_TARGET = -2;
-
-    /*
-     * public float savedXaxis = DEFAULT_PAN; public float savedYaxis = DEFAULT_TILT;
-     */
-    public TargetObject getVisionDataByTarget(int target) { // put in -1 for center target, -2 for
-                                                            // right and -3 for left
-        ArrayList<TargetObject> pr = getVisionData();
-        Collections.sort((List<TargetObject>) pr);
-        // pr should now be sorted
-
-        if (pr.size() == 0) {
-            if (PRINT_STUFF)
-                System.out.println("Can't find a target");
-            return null;
-        }
-        if (target == CENTER_TARGET) {
-            if (pr.size() % 2 == 0)
-                return pr.get(pr.size() / 2);
-            else
-                return pr.get((int) (pr.size() / 2 + 0.5));
-        } else if (target == RIGHT_TARGET)
-            return pr.get(pr.size() - 1);
-        else if (target == LEFT_TARGET)
-            return pr.get(0);
-        else
-            return pr.get(target);
     }
 
-    public float GetAspectRatio() {
-        return cachedTarget.aspectRatio;
-    }
-
-    public float GetBoundingArea() {
-        return cachedTarget.boundingArea;
+   public Camera(String ip) {
+        super();
+        RPi_addr = ip;
     }
 
-    public boolean HasTarget() {
-        return cachedTarget == null ? false : true;
-    }
+   public void GetTargets() {
+       ArrayList<TrackerBox2.TargetObject> targets = trackerbox.getVisionData();
+              
+       if (targets != null) {
+           System.out.println("I found "+targets.size()+" targets.");
```

Comment:
the PRINT_STUFF boolean is so we don't get the prints if we don't want them, maybe add an if(PRINT_STUFF) before the system outs?